packages/base/lib/redis.js

```diff
@@ -7,4 +7,7 @@ bluebird.promisifyAll(redis.Multi.prototype)
 
 const client = redis.createClient(REDIS_URL)
 
+client.__defaultExpire = 3 * 7 * 24 * 60 * 60 // 3 weeks
```

Comment:
Maybe work in "unit", e.g. `client.__defaultExpireSecs` or `client.__defaultExpirationInSeconds`